lua/catppuccin/groups/integrations/feline.lua

```diff
@@ -305,6 +305,24 @@ function M.get()
 		},
 		left_sep = invi_sep,
 	}
+
+	-- lazy.nvim updates
+	components.active[1][14] = {
+		provider = require("lazy.status").updates,
```

Comment:
I guess this line is calling lazy.nvim no matter the `enabled` function returns true or false, once this file is loaded.

As you can see below, the 'workspace loader' module, the `provider` is also a function so that it won't call something that doesn't exist.

Reply:
Ok the provider is also done with a protected call. It works as expected.

Also I update the deprecated api functions with the last commit.